src/l500/ac-trigger.cpp

```diff
@@ -210,11 +210,18 @@ namespace ivcam2 {
         if( _autocal->_to_profile )
         {
             // Reset the calibration so we can do it all over again
-            if( auto color_sensor = _autocal->_dev.get_color_sensor() )
+            if (auto color_sensor = _autocal->_dev.get_color_sensor())
+            {
                 color_sensor->reset_calibration();
+                AC_LOG(INFO, "Calibration data was reseted");
```

Comment:
was reset
(no ed)

Reply:
Why not put this message after line 219?

Reply:
Will be updated